packages/cli/src/collect/collect.js

```diff
@@ -108,7 +108,9 @@ async function startServerAndDetermineUrls(options) {
       const {child, patternMatch, stdout, stderr} = await runCommandAndWaitForPattern(
         options.startServerCommand,
         regexPattern,
-        {timeout: 10000}
+        {
```

Comment:
revert this one?

Reply:
This happen with linter (—fix) but es I can revert that